### DISCUSSION TITLE
Android v3.117.3 - Refresh Session login UI won't keep focus on 'Password' field

### DISCUSSION THREAD

**Discussion by dennisstewart** [standardnotes/forum#2915] [General]
**Describe the bug**
I updated my encryption on the web app. Upon opening the Android app, I am prompted to enter my credentials to refresh my session with the server.

After entering a character in the Password field, focus switches to the Email field so the next character typed will be placed there. The same goes for backspacing a character or pasting multiple characters. This issue occurs using Gboard or Samsung Keyboard. See video I recorded below.

**To Reproduce**
1. Go to Refresh Session form
2. Click on 'Password'
3. Enter any character on the keyboard or past text
4. Focus shifts to 'Email'

**Expected behavior**
Focus remains on the 'Password' field.

**Video**
https://user-images.githubusercontent.com/7216549/205712824-f13f8830-45d9-4e7f-9524-ab3b638ff18f.mp4

**Smartphone:**
 - Device: Samsung Galaxy S10
 - OS: Android 12 (One UI 4.1), Google Play system update September 1. 2022
 - Keyboard: Gboard 12.3.04.475639976-release-arm64-v8a, Samsung Keyboard 5.4.80.1
 - Standard Notes Version 3.117.3

**Additional context**
I'm fine pasting from my password manager, but entering a long passphrase by hand is a pain in the neck with this bug.

**Comment by effieeee** [comment]
Hi @dennisstewart, thanks for including the screen recording with your bug report. I'll see if we can reproduce this and look into the issue further, though are you also encountering this authentication prompt every time that the Android app is launched now? If so, that shouldn't be the case. 

Can I ask you to clear the app data as instructed [here](https://docs.standardnotes.com/troubleshooting/reset-apps/#android) so that the app starts from its default state, then try signing in again?

**Comment by dennisstewart** [comment]
It was every time I opened it after updating the encryption. I no longer get prompted after entering my credentials once (copied from password manager instead of typing the passphrase).
